onboarding.md

```diff
@@ -0,0 +1,53 @@
+# Ocean Transport Group Onboarding Checklist
+
+All new group members should go through the following steps.
+
+## Expectations and Code of Conduct
+
+- [ ] Review the group [participation guide](https://ocean-transport.github.io/guide.html) and make
+      sure you understand your expectations.
```

Comment:
I am not clear what 'your expectations' means here. Expectations towards a new member or the member's expectation towards other folks?